packages/app/app/components/ListeningHistoryView/testSuite.js

```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+import { render, screen, fireEvent } from '@testing-library/react';
+import { PieChartTopArtists } from './PieChartTopArtists';
+
+const mockData = [
+  { name: 'Artist A', count: 50 },
+  { name: 'Artist B', count: 40 },
+  { name: 'Artist C', count: 30 },
+  { name: 'Artist D', count: 20 },
+  { name: 'Artist E', count: 10 }
+];
+
+describe('PieChartTopArtists', () => {
+  test('renders without crashing', () => {
```

Comment:
This can be replaced with a snapshot test, which gives us the same information, and also records what the component looks like in the HTML.